contracts/homa/Homa.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.0;
+
+import "./IHoma.sol";
+
+contract Homa is IHoma {
+    address constant private precompile = address(0x0000000000000000000000000000000000000406);
```

Comment:
change to `0x0000000000000000000000000000000000000407` and add here.
https://github.com/AcalaNetwork/Acala/blob/59339f95690e6738e6e8a3bb2ae11be1eedb33d4/runtime/common/src/precompile/mod.rs#L80

Reply:
https://github.com/AcalaNetwork/Acala/blob/59339f95690e6738e6e8a3bb2ae11be1eedb33d4/runtime/common/src/precompile/mod.rs#L175